device/common/include/traccc/finding/device/propagate_to_next_surface.hpp

```diff
@@ -41,6 +42,7 @@ TRACCC_DEVICE inline void propagate_to_next_surface(
     typename propagator_t::detector_type::view_type det_data,
     bfield_t field_data,
     bound_track_parameters_collection_types::const_view in_params_view,
+    vecmem::data::vector_view<const unsigned int> param_ids_view,
```

Comment:
```suggestion
    const vecmem::data::vector_view<const unsigned int>& param_ids_view,
```

I'm very happy that you correctly used the `const unsigned int` template parameter. Now we should make the variable itself `const` as well. :wink:

This should generally be cleaned up in all of these functions a bit, but I'm happy with just setting this one variable correctly as a start as well. :smile:

Reply:
OK